tlg0012.tlg001.perseus-eng4.pope-alignments:1.25-1.29.py

```diff
@@ -0,0 +1,13 @@
+alignments = {
+    "1.25–1.29": {
+        "butler": """who spoke fiercely to him and sent him roughly away. "Old man," said he, "let me not find you tarrying about our ships, nor
+                  yet coming hereafter. Your scepter of the god and your wreath shall profit you nothing. I will not free her. She shall grow old""",
+        "pope": """
+        May Jove restore you when your toils are o'er
```

Comment:
I'll go ahead and merge this in, but I just want to flag that although this might be where line 25 starts in the Pope, it doesn't seem to line up with the Butler -- the line numbers won't actually match, but we want to make sure that the _sense_ matches.